### ISSUE TITLE
new editorials for 30 days of JS

### ISSUE THREAD

**Issue by DebajyotiX** [akhilkammila/leetcode-screenshotter#1]
https://leetcode.com/studyplan/30-days-of-javascript/
This learning module has 30 questions, most of them has hidden Editorials(premium only).
Can you add these Editorials. I am pasting the question IDs here:
[2667,
2620,
2704,
2665,
2635,
2634,
2626,
2629,
2703,
2666,
2623,
2723,
2621,
2715,
2725,
2637,
2622,
2627,
2721,
2727,
2677,
2619,
2631,
2724,
2722,
2625,
2705,
2694,
2695,
2726,
]

**Comment by akhilkammila** [comment]
So I only did the ones under the "algorithms" problemset (https://leetcode.com/problemset/algorithms/) cuz that's what I was using leetcode for.

I can do the other probs tho. Are you doing the 30 days of JS thing, or did you just notice that was missing? I can get the screenshots up in the next few weeks are so depending on when I'm free.

**Comment by DebajyotiX** [comment]
Yeah, I am actually doing that JS 30 Days challenges. It will be helpful to read those editorials after solving the problems. Whenever you are free.  If you need any help with fetching or organising the problem list data, I will be at your service.

**Comment by shivam2003sy** [comment]
If you can provide! it will be of great help  thanks

**Comment by sg-prashanth-raju** [comment]
@akhilkammila When will the editorials be added for these problems?
